examples/2d/many_sprites.rs

```diff
@@ -90,7 +90,7 @@ fn tick(time: Res<Time>, sprites: Query<&Sprite>, mut query: Query<&mut PrintTim
         timer.0.tick(time.delta());
 
         if timer.0.just_finished() {
-            println!("Sprites: {}", sprites.iter().count(),);
+            warn!("Sprites: {}", sprites.iter().count(),);
```

Comment:
Hm, not sure I see why `warn` and not `info`?

Reply:
in this example, the `LogDiagnosticsPlugin` plugin is enabled with `FrameTimeDiagnosticsPlugin`, which adds 3 lines of `INFO` log per second. As this one also logs every second, a `warn` helps it to stand out

Reply:
I'm not sure we should use `warn` just to increase visibility among other "noisy" logs. That encourages users to create logs in ways that don't line up with their semantics.

Reply:
back to `info` it is!